exasol/exaslpm/pkg_mgmt/install_apt.py

```diff
@@ -0,0 +1,71 @@
+import subprocess
+from typing import List
+
+from exasol.exaslpm.model.package_file_config import AptPackages
+
+
+def prepare_update_command() -> list[str]:
+    update_cmd = ["apt-get", "-y", "update"]
+    return update_cmd
+
+
+def prepare_clean_cmd() -> list[str]:
+    clean_cmd = ["apt-get", "-y", "clean"]
+    return clean_cmd
+
+
+def prepare_autoremove_cmd() -> list[str]:
+    autoremove_cmd = ["apt-get", "-y", "autoremove"]
+    return autoremove_cmd
+
+
+def prepare_ldconfig_cmd() -> list[str]:
+    ldconfig_cmd = ["ldconfig"]
+    return ldconfig_cmd
+
+
+def prepare_locale_cmd() -> list[str]:
+    locale_cmd = ["locale-gen", "&&", "update-locale", "LANG=en_US.UTF8"]
+    return locale_cmd
+
+
+def prepare_install_cmd(apt_packages: AptPackages) -> list[str]:
+    install_cmd = ["apt-get", "install", "-V", "-y", "--no-install-recommends"]
+    if apt_packages.packages is not None:
+        for package in apt_packages.packages:
+            install_cmd.append(f"{package.name}={package.version}")
+    return install_cmd
+
+
+class CommandExecutor:
+    def execute(self, command: list[str]):
+        print(f"Executing: {command}")
+        result = subprocess.run(command, capture_output=True)
+        print(
```

Comment:
I think, we should return an object that, we can ask for the returncode of the command and a stream of stdout and stderr. Probably, a wrapper around the return value of the subprocess.run call.